docs/fast_data/configuration/single_view_creator/index.md

```diff
@@ -66,6 +66,16 @@ We strongly recommend using the plugin. The template is supposed to be used only
 | KAFKA_CONSUMER_MAX_WAIT_TIME_MS     | -        | (v6.2.1 or higher) The maximum amount of time in milliseconds the server will block before answering the fetch request if there isn't sufficient data to immediately satisfy the requirement given by minBytes [1 byte]                                                                                                                                                                                                                                                                                                     | 500                 |
 | SV_UPDATE_VERSION                   | -        | (v6.2.1 or higher) Define which version of the `sv-update` event should be emitted by the service. Accepted values are `v1.0.0` and `v2.0.0`. By default, for retro-compatibility, version `v1.0.0` is employed                                                                                                                                                                                                                                                                                                             | v1.0.0              |
+| CONTROL_PLANE_ACTIONS_TOPIC         | -        | Topic where the Single View Creator expects to receive the action commands of `pause` and `resume`.                                                                                                                                                                                                                                                                                                                                                                                                                         | -                   |
+| CONTROL_PLANE_KAFKA_GROUP_ID        | -        | The Kafka Consumer Group Identifier for the action commands client.                                                                                                                                                                                                                                                                                                                                                                                                                                                         | -                   |
+
+
+:::note
+
+Setting up both the variable `CONTROL_PLANE_ACTIONS_TOPIC` and `CONTROL_PLANE_KAFKA_GROUP_ID` enables the communication between the Single View Creator and the [Runtime Management](/fast_data/runtime_management.mdx). This means that the Single View Creator will receive and execute the commands from the latter.
```

Comment:
Do we need to mention that SVCs won't process any record till the Runtime Management notifies them to start/resume processing?

Reply:
Thank you for the suggestions. I have directly added a section `Runtime Management` in the Single View Creator index.js page. I also specified from which version we support it, both in the SVC page and the Control Plane one.